doc/source/visualizing/plots.rst

```diff
@@ -769,29 +769,98 @@ two element tuples.
    slc.set_center((0.5, 0.503))
    slc.save()
 
-Flipping the plot view axes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adjusting the plot view axes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 By default, all :class:`~yt.visualization.plot_window.PlotWindow` objects plot
 with the assumption that the eastern direction on the plot forms a right handed
 coordinate system with the ``normal`` and ``north_vector`` for the system, whether
-explicitly or implicitly defined. This setting can be toggled or explicitly defined
-by the user at initialization:
+explicitly or implicitly defined. There are a number of ways the orientation of
+the view can be adjusted.
+
+The first two axis orientation modifications,
+:meth:`~yt.visualization.plot_window.AxisAlignedSlicePlot.flip_horizontal`
+and :meth:`~yt.visualization.plot_window.AxisAlignedSlicePlot.flip_vertical`, are
+equivalent to the ``invert_xaxis`` and ``invert_yaxis`` of matplotlib ``Axes``
+objects. ``flip_horizontal`` will invert the plot's x-axis while the :meth:`~yt.visualization.plot_window.AxisAlignedSlicePlot.flip_vertical` method
+will invert the plot's y-axis
 
 .. python-script::
 
    import yt
 
    ds = yt.load("IsolatedGalaxy/galaxy0030/galaxy0030")
-   # slicing with non right-handed coordinates
-   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"), right_handed=False)
-   slc.annotate_title("Not Right Handed")
+   # slicing with standard view (right-handed)
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"))
+   slc.annotate_title("Standard Horizontal (Right Handed)")
+   slc.save("Standard.png")
+
+   # flip the horizontal axis (not right handed)
+   slc.flip_horizontal()
+   slc.annotate_title("Horizontal Flipped (Not Right Handed)")
    slc.save("NotRightHanded.png")
 
-   # switching to right-handed coordinates
-   slc.toggle_right_handed()
-   slc.annotate_title("Right Handed")
-   slc.save("Standard.png")
+   # flip the vertical axis
+   slc = yt.SlicePlot(ds, "x", ("gas", "velocity_x"), flip_vertical=True)
+   slc.annotate_title("Flipped vertical")
+   slc.save("FlippedVertical.png")
+
+Note that all three operations effectively toggle the plot's right-handedness.
```

Comment:
Ah, I'm sorry, I know that *I* suggested this sentence but I don't think it holds actually. What _does_ change is. he direction in which the normal vector is pointing. By default, it's pointing toward the reader, and applying an odd number of these operations will make it point away from the reader.

Right-handedness isn't actually affected by any of these operations. But more importantly, I think I just convinced myself that right-handedness was never a relevant concept for 2D plots, because in the absence of opacity (effectively what `ProjectionPlot` does), it is irrelevant whether you want to look at a transparent object from the front or the from the back.

Reply:
Will delete this line in the next commit!